web/app/templates/components/map-reduce-exception.hbs

```diff
@@ -29,7 +29,9 @@
                     {{#p.toggle}}
                         <p>{{application.name}}</p>
                     {{/p.toggle}}
-
+                   <div class="box">
```

Comment:
From the [screenshot](https://user-images.githubusercontent.com/16147255/79328806-4328dd80-7f34-11ea-93c4-7a30740d83ee.png) looks like the scaling for the div is not proportionate to the divs below, can you look into it?

Reply:
Since users is expected to see exceptions from AZ  UI not from Dr E , this cosmetic changes is of minor importance .  This can be taken up in next PR , if required